src/main/java/org/dataloader/registries/DispatchPredicate.java

```diff
@@ -0,0 +1,92 @@
+package org.dataloader.registries;
+
+import org.dataloader.DataLoader;
+
+import java.time.Duration;
+import java.util.Objects;
+
+/**
+ * A predicate class used by {@link ScheduledDataLoaderRegistry} to decide whether to dispatch or not
+ */
+@FunctionalInterface
+public interface DispatchPredicate {
```

Comment:
This is very similar to `Predicate`, why not use that?

Reply:
because it takes 2 arguments - so inspired by but not the same